README.md

```diff
@@ -1,2 +1,143 @@
 # To-do-list
-"To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+
+- [🚀 Live Demo](#live-demo)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 <a name="about-project">TO DO LIST</a>
+
+This repository contains the source codes for a To Do List. It showcases my skills and expertise in creating Objects with Classes, Sectioning with Modules and using Add Event Listeners. I used HTML, CSS and Javascript programming languages in the creation of this project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.css3.com/">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Functionality: The navigation bar to make it easy for users to navigate the website**
+- **List: A clicakable link on the navigation bar that displays the books inputted by the user**
+- **Add New:A clickable link that displays a form allowing users to input the title and author of books**
+- **Social Contact: A form allowing visitors to get in touch with me.**
+- **Responsive Design: The website is optimized for different devices and screen sizes.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
```

Comment:
- [ ] Kindly remove the `Live Demo` heading in the Readme file, since you haven't deployed the project. Currently, the link is not directing to the deployed project, kindly remove it in order to have a professional Readme file